server_addon/houdini/server/settings/general.py

```diff
@@ -0,0 +1,39 @@
+from pydantic import Field
+from ayon_server.settings import BaseSettingsModel
+
+
+class HoudiniVarModel(BaseSettingsModel):
+    _layout = "expanded"
+    var: str = Field("", title="Var")
+    value: str = Field("", title="Value")
+    is_dir_path: bool = Field(False, title="is Dir Path")
```

Comment:
Title seems a bit off with the `Dir Path` capitalized but `is` being lowercase? Should this be `Is dir path` or if the title is the descriptive label probably "Value should be treated as a path" would be nicer for the setting?

Also, why do you feel `is_dir_path` is clearer than `is_path`? Technically one _could_ point it to a file right? `{asset}_thumbnail.png` or whatever.

Reply:
>  Technically one could point it to a file right? {asset}_thumbnail.png or whatever.

yes, That's why I though `is_dir_path` is clearer 

> Title seems a bit off with the Dir Path capitalized but is being lowercase? 

I was going to write it as `isDirPath`

Reply:
Just to rephrase what I said. The value can point to a file just fine - it isn't necessarily a directory - technically someone COULD point it to a `{asset}_thumbnail.png` **but** thinking about it now that'd definitely give odd behavior if it were to CREATE that as a  folder if it didn't exist ha so I see what you're getting at. Thanks.

Anyway, I still think "Treat value as a director (create if it does not exist)" is the most clear. "Is folder" or "Is directory" doesn't tell me enough about what the checkbox would be doing. Right?

Reply:
"Treat value as a directory, create directory if it does not exist" is very long! 
is this clear? I wish if we could make it shorter, 
![image](https://github.com/ynput/OpenPype/assets/20871534/082a5b9b-3b81-4f8e-ae06-92c13a2cb368)

or `Is value a directory` 
![image](https://github.com/ynput/OpenPype/assets/20871534/fdad826e-48f9-423b-8243-48bfeb8d5430)

Reply:
**Treat as directory** is short and gets the point across. Would that work for you?


I just wish there was a label, tooltip or description the user could rely on. Maybe at the top of the section there could be a label that says: "If a value is treated as a directory on update it will be ensured the folder exists" or alike.

Reply:
> Treat as directory 

That's fair enough 

> I just wish there was a label, tooltip or description the user could rely on.

I think this is supported in Ayon, but I'm not sure how to use it (yet). 

in openpype we can just insert a label as tip.
![image](https://github.com/ynput/OpenPype/assets/20871534/19322669-8c72-4429-b26d-a61d45ee7456)

Reply:
I personally quite like this!

Reply:
omg! you just add a doc string!  and click `?` button to show it! 

![image](https://github.com/ynput/OpenPype/assets/20871534/d6f039d9-b4f7-4660-9fd6-a4823f8d283d)

![image](https://github.com/ynput/OpenPype/assets/20871534/8eb4da68-cacb-48bd-ad67-939b1ef40b16)

Reply:
But how do you set a tooltip for a single field?
Still, this is already nice for this particular use case.